README.md

```diff
@@ -56,10 +56,10 @@ VERSION
 ```
 
 A valid Fastly API `-token` is mandatory. [See this link][token] for information
-on creating API tokens. 
+on creating API tokens. Token can be provided via environment variable `FASTLY_API_TOKEN`.
```

Comment:
> The token can also be provided via the `FASTLY_API_TOKEN` environment variable.

And please wrap at 80col.

Reply:
Fixed in https://github.com/peterbourgon/fastly-exporter/pull/18/commits/29b472ab3c90bc5d05b87a269f7b8d25338856e1